assets/src/editor/events/dates-and-times/editor-date/filter-bar/index.js

```diff
@@ -2,31 +2,41 @@
  * External imports
  */
 import { withEntityPagination } from '@eventespresso/higher-order-components';
+import { compose, createHigherOrderComponent } from '@wordpress/compose';
 
 /**
  * Internal dependencies
  */
-import withDatesListFilterBar from './with-dates-list-filter-bar';
+import withDateEntitiesListFilterBar, {
+	getFilteredDateEntitiesList,
+} from './with-date-entities-list-filter-bar';
 import withDatesListFilterState from './with-dates-list-filter-state';
 
-const PaginatedDatesListWithFilterBar = (
-	EditorDates,
-	paginationConfig = {},
-) => withDatesListFilterBar(
-	withEntityPagination( paginationConfig )( EditorDates )
+const withPaginatedDateEntitiesListWithFilterBar = (
```

Comment:
`PaginatedDatesListWithFilterBar` wasn't  usable in a typical HOC chain if you wanted to pass in a custom `paginationConfig` object (unless you wrapped it in another component.  This change here makes it a curried function so custom `paginationConfig` can be passed in easier.